### PR TITLE
add a local function test case

### PR DESCRIPTION
**Pull request by liqunfu**
**Description** 
add a local function test case

- Describe your changes.
refactor make_function helper method
add test case for models with local functions

**Motivation and Context**
- improve runtime test coverage of onnx local function specification

### CODE REVIEW

**Comment by askhade** [comment]
@liqunfu : DCO check is failing for this PR because the commits are not signed. including "-s" option in every git commit cmd enables sign off. 
```
git commit -s -m "my commit"
```

You can check your local logs to make sure the commit is signed. If you push more than 1 unsigned commits there is no way to go back and sign the only option I know is to resend a new PR. Unfortunately, this is the current limitation.

File: onnx/backend/test/case/model/local-functions.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+    @staticmethod
+    def export():  # type: () -> None
+
+        ops = [
+            helper.make_opsetid("", 10),
+            helper.make_opsetid("local", 10)
+        ]
+
+        g = GraphProto()
```
Comment (askhade) [inline onnx/backend/test/case/model/local-functions.py:30]: I suggest using onnx parser api for test model creation. It is more readable and easier to use.

**Comment by askhade** [comment]
Can you add details around what exactly this test covers.

**Comment by liqunfu** [comment]
close this pr.
Local function is better tested be runtime (e.g. https://github.com/microsoft/onnxruntime/blob/master/onnxruntime/test/ir/onnx_model_test.cc).

**Pull request closed without merging**